common-content/en/module/decomposition/backend-limitations/index.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Limitations of backends"
+headless = true
+time = 20
+facilitation = false
+emoji= "📖"
+objectives = [
+    "Explain why a backend on its own doesn't provide reliable data persistence.",
+    "Explain why we may prefer doing work in a frontend rather than backend to avoid latency.",
+]
++++
+
+We've already explored limitations of frontends.
+
+We know a backend is just a program that runs for a long time.
+
+### Lifetime
+
+A major limitation of backends is that "a long time" probably isn't forever.
+
+Sometimes we change the code of the backend. Or need restart the computer it's running on for an upgrade. Or its computer loses power and we need to start it again.
```

Comment:
```suggestion
Sometimes we change the code of the backend. Or we need to restart the computer it's running on for an upgrade. Or its computer loses power and we need to start it again.
```

Suggest bringing in something from cyf+ eg computers actually exist in the world.